README.md

```diff
@@ -1,49 +1,72 @@
+  </a>
+</p>
+
+---
 
-[libSQL](https://turso.tech/libsql) is an open source, open contribution fork of SQLite, created and maintained by [Turso](https://turso.tech). We aim to evolve it to suit many more use cases than SQLite was originally designed for, and plan to use third-party OSS code wherever it makes sense.
+## Documentation
 
-libSQL is licensed under an [Open Source License](LICENSE.md), and we adhere to a clear [Code of Conduct](CODE_OF_CONDUCT.md)
+We aim to evolve it to suit many more use cases than SQLite was originally designed for, and plan to use third-party OSS code wherever it makes sense. 
 
-## Features
+libSQL has many great features, including:
 
 * Embedded replicas that allow you to have replicated database inside your app.
 * [libSQL server](libsql-server) for remote SQLite access, similar to PostgreSQL or MySQL
 * Supports Rust, JavaScript, Python, Go, and more.
 
 There are also various improvements and extensions to the core SQLite:
 
-* [`ALTER TABLE` extension for modifying column types and constraints](https://github.com/libsql/libsql/blob/main/libsql-sqlite3/doc/libsql_extensions.md#altering-columns)
-* [Randomized ROWID](https://github.com/libsql/libsql/issues/12)
-* [WebAssembly User Defined Functions](https://blog.turso.tech/webassembly-functions-for-your-sqlite-compatible-database-7e1ad95a2aa7)
-* [Pass down SQL string to virtual table implementation](https://github.com/libsql/libsql/pull/87)
-* [Virtual write-ahead log interface](https://github.com/libsql/libsql/pull/53)
+* [`ALTER TABLE` extension for modifying column types and constraints](https://github.com/tursodatabase/libsql/blob/main/libsql-sqlite3/doc/libsql_extensions.md#altering-columns)
+* [Randomized ROWID](https://github.com/tursodatabase/libsql/issues/12)
+* [WebAssembly User Defined Functions](https://turso.tech/blog/webassembly-functions-for-your-sqlite-compatible-database-7e1ad95a2aa7)
+* [Pass down SQL string to virtual table implementation](https://github.com/tursodatabase/libsql/pull/87)
+* [Virtual write-ahead log interface](https://github.com/tursodatabase/libsql/pull/53)
 
 The comprehensive description can be found [here](libsql-sqlite3/doc/libsql_extensions.md)
 
-## Getting Started
+### Official Drivers
 
-The project provides two interfaces: the libSQL API, which supports all the features, and the SQLite C API for compatibility.
-
-To get started with the libSQL API:
-
-* [JavaScript](https://github.com/libsql/libsql-experimental-node)
+* [TypeScript / JS](https://github.com/tursodatabase/libsql-client-ts)
 * [Rust](libsql) 
-* [Python](https://github.com/libsql/libsql-experimental-python) (experimental)
-* [Go](https://github.com/libsql/go-libsql)
+* [Go](https://github.com/tursodatabase/go-libsql)
+
+### Experimental Drivers
+* [Python](https://github.com/tursodatabase/libsql-experimental-python) (experimental)
```

Comment:
Go also has one experimental driver

Reply:
oh, so `go-libsql` was experimental earlier. I see that it is renamed. So, we need to mention the other driver - https://github.com/tursodatabase/libsql-client-go?